google/cloud/datastore/key.py

```diff
@@ -549,12 +614,14 @@ def _get_empty(value, empty_value):
 def _check_database_id(database_id):
```

Comment:
Consider inline the function. (only 1 usage AFAIC)

Reply:
I think I'll leave it as-is for now for consistency's sake.